kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js

```diff
@@ -53,50 +53,51 @@ define([
                 value: config.initialValue,
                 disabledValues: new Set(config.disabledValues || []),
                 invalidValues: new Set(config.invalidValues || []),
-            };
+            },
+            devMode = config.devMode || false;
+        let useMultiselect = false;
+        try {
+            if (spec.original.dropdown_options.multiselection) {
+                useMultiselect = true;
+            }
```

Comment:
the magical toggle!

Reply:
Once this is merged, etc., I'll edit the kb-sdk docs to include info about this multi select option so people don't have to put up with having numerous dropdowns.